course/src/main/java/org/openedx/course/presentation/dates/CourseDatesFragment.kt

```diff
@@ -163,6 +163,10 @@ class CourseDatesFragment : Fragment() {
         }
     }
 
+    fun updateData() {
```

Comment:
Maybe we can use notifiers instead.

Reply:
we are following the implemented structure accordingly.